src/components/Header/Header.tsx

```diff
@@ -50,16 +53,114 @@ const HomepageLink = styled(Link)`
   }
 `
 
-const jobLinkMobile = css`
-  ${from.desktop} {
-    display: none;
-  }
+const socialLinksWrapper = css`
+  display: flex;
+  left: 0;
+  position: absolute;
+  top: ${space[4]}px;
 `
 
+const socialLinkWrapper = css`
+  margin-left: ${space[2]}px;
+`
+
+const socialButton = {
+  button: {
+    textPrimary: colors.primaryText,
+    backgroundPrimary: colors.silverGrey
+  }
+}
+
+// TODO: Add this to Source?
+const SvgRSS = () => {
+  return (
+    <svg
+      css={css`
+        padding: ${space[1]}px;
+      `}
+      width="18"
+      height="18"
+      viewBox="0 0 18 18"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        id="Vector"
+        fillRule="evenodd"
+        clipRule="evenodd"
+        // eslint-disable-next-line max-len
+        d="M0 0V3.27378C8.13239 3.27378 14.7263 9.86761 14.7263 18H18C18 8.06298 9.93702 0 0 0ZM2.52185 12.9563C1.12907 12.9563 0 14.0853 0 15.4781C0 16.871 1.12907 18 2.52185 18C3.91463 18 5.0437 16.871 5.0437 15.4781C5.0437 14.0853 3.91463 12.9563 2.52185 12.9563ZM0 9.82134V6.54756C6.32776 6.54756 11.4525 11.6722 11.4525 18H8.17866C8.17866 13.4884 4.51157 9.82134 0 9.82134Z"
+        fill="black"
+      />
+    </svg>
+  )
+}
+
+// TODO: Add this to Source?
+const SvgGithub = () => {
```

Comment:
1. I extracted SvgRSS path from the github's homepage source
Is it something we would like to move into the Source? Happy to do that, if so
cc @SiAdcock

2. I had issues reverting the colours to a grey background and black fill. @mxdvl if you have some insights on how to do that it would be great :)

Reply:
Not sure what you mean by reverting the colours?

I’ve removed the width and height values from the SVGs as they are set via css already.

Reply:
Oops Sorry typo, I meant invert, the figma styles show it with grey background and black body but when I tried to do that using `fill` it didn't work out don't know why

Reply:
Ahh, gotcha, you would have to do some knock-out effect. I think that's a fix for later, as you would also need the SVG to stretch across the whole button, which it cannot currently do from Source.